src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -62,7 +61,7 @@ See the [Amazon Cognito documentation]( https://docs.aws.amazon.com/cognito/late
 Once you have chosen your management method you can configure your Auth category and install the library to use with your application.
```

Comment:
Small tweaks:

Once you have chosen how to set up your backend resources you can configure your Auth category and install the Amplify libraries to use with your application.